distribution/docs/src/main/resources/_contents/_apis/application-service-api-contents.adoc

```diff
@@ -0,0 +1,131 @@
+Here are the interfaces that are exposed in the Managed Bean:
+
+getApplicationTree:: Creates an application hierarchy tree that shows relationships between applications.
+startApplication:: Starts an application with the given name.
+stopApplication:: Stops an application with the given name.
+addApplications:: Adds a list of application that are specified by their URL.
+
+==== Configuration Files
+
+Support for configuration files was added to allow for an initial installation of applications on first run.
+
+==== Initial Application Installation
+
+[WARNING]
+====
+This application list configuration file is only read on first start.
+====
+
+To minimize the chance of accidentally installing and uninstalling applications, the configuration file for installing the initial applications is only read the first time that ${branding} is started.
+The only way to change what applications are active on startup is to use the console commands.
+Operations can also be done with the administrator web console that comes with ${branding} using the Features tab and installing the main feature for the desired application.
+
+The application list file is located at `${ddf-branding}_HOME/etc/org.codice.ddf.admin.applicationlist.properties`
+
+Applications should be defined in a `<name>=<format>` syntax where location may be empty for applications that have already been added to ${branding} or were prepackaged with the distribution.
+
+.Examples:
+[source]
+----
+# Local application:
+opendj-embedded
+
+# Application installed into a local maven repository:
+opendj-embedded=mvn:org.codice.opendj.embedded/opendj-embedded-app/1.0.1-SNAPSHOT/xml/features
+
+# Application located on the file system:
+opendj-embedded=file:/location/to/opendj-embedded-app-1.0.1-SNAPSHOT.kar
+----
+
+Applications will be started in the order they are listed in the file.
+If an application is listed, ${branding} will also attempt to install all dependencies for that application.
+
+==== Application Service Interfaces
+
+. `ApplicationService` 
+. `Application`
+. `ApplicationStatus`
+. `ApplicationNode`
+
+===== `ApplicationService` Interface
+
+The ApplicationService interface is the main class that is used to operate on applications.
+
+`getApplications`:: This method returns a set of all applications that are installed on the system. Callers can then use the Application handle to get the name and any underlying features and bundles that this application contains.
+`getApplications`:: Returns the application that has the given name.
+`startApplication`:: Starts an application, including any defined dependencies in the application.
+`stopApplication`:: Stops an application, does not include any external transitive dependencies as they may be needed by other applications.
+`addApplication`:: Adds a new application to the application list. *NOTE: This does NOT start the application.*
+`removeApplication`:: Removes an application that has the given URI.
```

Comment:
Should add a note here as well about whether the application is stopped first if it is running or if this can't be run against an application that is running. Which information I see is missing from the javadocs so I couldn't say without further investigation what the behavior is.